activesupport/test/core_ext/duration_test.rb

```diff
@@ -46,6 +47,11 @@ def test_to_s
     assert_equal "1", 1.second.to_s
   end
 
+  def test_in_milliseconds
+    assert_equal 86400000, 1.day.in_milliseconds
+    assert_equal 60000, 1.minute.in_milliseconds
```

Comment:
`in_seconds` is arguably the one that's [already] wrong and at fault here, but:

```suggestion
    assert_equal 60000, 1.minute.in_milliseconds
    assert_equal 1500, 1.5.seconds.in_milliseconds
```

Reply:
Hmm, maybe `in_seconds` should just return `@value`? Especially if we're going to keep `to_f`, `to_r`, etc.